src/live.rs

```diff
@@ -280,6 +268,19 @@ fn write_live_iso(iso: &IsoConfig, input: &mut File, output_path: Option<&String
     Ok(())
 }
 
+fn write_live_pxe(initrd: &Initrd, output_path: Option<&String>) -> Result<()> {
```

Comment:
Minor: might be worth a docstring for this mentioning that we assume that in the stdout case, the caller already verified that it's not a TTY.